docs/source/descriptions/shapefilepreparation.rst

```diff
@@ -0,0 +1,138 @@
+Shapefile preparation
```

Comment:
I would avoid a sub-package for this in the docs. Maybe just use something like `prepare.rst`?